pisp_app/lib/Screens/new_payment.dart

```diff
@@ -0,0 +1,114 @@
+import 'package:flutter/material.dart';
+import 'package:pispapp/MockData/account.dart';
+import 'package:pispapp/Screens/lookup_payee.dart';
+import 'package:pispapp/Screens/phone_number.dart';
+
+class NewPayment extends StatefulWidget {
+  @override
+  _NewPaymentState createState() => _NewPaymentState();
+}
+
+class _NewPaymentState extends State<NewPayment> {
+  _NewPaymentState() {
+    final List<Account> accounts = getMyDummyAccounts();
+    _accounts = accounts.where((Account element) => element.linked).toList();
+  }
+  
+  List<Account> _accounts = <Account>[];
+  Account selectedAccount;
+
+  String phoneNumber;
+  String phoneIsoCode;
+
+  void onPhoneNumberChange(
+      String number, String internationalizedPhoneNumber, String isoCode) {
+    setState(() {
+      phoneNumber = number;
+      phoneIsoCode = isoCode;
+    });
+  }
+
+  
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+        child: Column(
+      children: <Widget>[
+        const SizedBox(height: 25),
+        const Text(
+          'Choose an account',
+          style: TextStyle(
+            fontSize: 18.0,
+          ),
+        ),
+        const SizedBox(height: 16),
+        DropdownButton<Account>(
+          hint: const Text('Select Account'),
+          value: selectedAccount,
+          items: _accounts.map((Account account) {
+            return DropdownMenuItem<Account>(
+              value: account,
+              child: Row(
+                children: <Widget>[
+                  Text(
+                    account.alias,
+                    style: TextStyle(color: Colors.black),
+                  ),
+                ],
+              ),
+            );
+          }).toList(),
+          onChanged: (Account account) {
+            setState(() {
+              selectedAccount = account;
+            });
+          },
+        ),
+        const SizedBox(height: 60),
+        const Text(
+          'Payee Phone Number',
+          style: TextStyle(
+            fontSize: 16.0,
+          ),
+        ),
+        const SizedBox(height: 16),
+        Padding(
+          padding: const EdgeInsets.fromLTRB(15.0, 0.0, 15.0, 0.0),
+          child: PhoneNumberInput(onPhoneNumberChange),
+        ),
+        Expanded(
```

Comment:
Instead of doing something like this, I think there's a widget that is specially designed to fit this purpose, which is [FloatingActionButton](https://api.flutter.dev/flutter/material/FloatingActionButton-class.html). You may need to wrap it inside a `Scaffold` tho. 

Besides, the button broke when I tried this...
![image](https://user-images.githubusercontent.com/28918289/85969544-8c001680-b9fa-11ea-88d4-96e209923bcf.png)

Reply:
I wanted to have a splash upon pressing the button. Can still be removed. What do you think ?

Reply:
I think you can still do that with `FloatingActionButton` (?) (not sure tho I never tried to do that before).
Besides, I think it is better to keep things simple and avoid bugs by using the built-in widgets.
But, yeah, if you still want to keep it like that, perhaps just need to fix the bugs when the keyboard is active.

Reply:
the button got squished there because there wasn't enough space between the keyboard and phone number widget. It should be fine now. I changed the divider height. Can you please take a look ?

Reply:
I'm going to resolve this conversation since you changed it to `FloatingActionButton`, thank you!